### PR TITLE
Community Gogrid

### PR DESCRIPTION
**Pull request by catlook**

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-d8a0416a3f/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-d55843ff5d/how-to/). :bow:

**Comment by cyrichardson** [comment]
@catlook Suggested edits here: 
[migrate-gogrid-cloud-servers-to-rackspace-cloud_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2780245/migrate-gogrid-cloud-servers-to-rackspace-cloud_cyrEdits.docx)

File: content/managed-operations/migrate-gogrid-cloud-servers-to-racksapce-cloud.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+#### Rackspace Service Level Agreements (SLA)
+
+The level of service provided at GoGrid aligns most closely with Rackspace's
+Managed Infrastructure SLA. For more details about the two SLAs offered by
+Rackspace Public Cloud, see [service levels](https://www.rackspace.com/openstack/public/service-levels).
+
+#### Rackspace regions
+
+Rackspace Public Cloud the following regions:
```
Comment (RackerKate) [inline content/managed-operations/migrate-gogrid-cloud-servers-to-racksapce-cloud.md]: "is available in?"

File: content/managed-operations/migrate-gogrid-cloud-servers-to-racksapce-cloud.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+Use the following steps to migrate your GoGrid Linux servers to the Rackspace
+Cloud:
+
+1. Sign up for a Rackspace Public Cloud account.
+
+2. Create a new destination server in the Rackspace Public Cloud and record
+   the public IPv4 address and root password of your new Rackspace server. See
+   [Create a general purpose Cloud Server](how-to/creating-a-general-purpose-cloud-server/)
+   for assistance. If your current servers are running an end-of-life (EOL)
+   operating system (OS), such as Ubuntu&reg; 12, contact Rackspace Support to
```
Comment (RackerKate) [inline content/managed-operations/migrate-gogrid-cloud-servers-to-racksapce-cloud.md]: Suggest removing comma after "(OS)."

File: content/managed-operations/migrate-gogrid-cloud-servers-to-racksapce-cloud.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+### Migration steps
+
+Use the following steps to migrate your GoGrid Linux servers to the Rackspace
+Cloud:
+
+1. Sign up for a Rackspace Public Cloud account.
+
+2. Create a new destination server in the Rackspace Public Cloud and record
+   the public IPv4 address and root password of your new Rackspace server. See
+   [Create a general purpose Cloud Server](how-to/creating-a-general-purpose-cloud-server/)
```
Comment (RackerKate) [inline content/managed-operations/migrate-gogrid-cloud-servers-to-racksapce-cloud.md]: This link didn't work in the preview. Might need to add an opening forward slash?

File: content/managed-operations/migrate-gogrid-cloud-servers-to-racksapce-cloud.md (hunk 1 above)
Comment (catlook) [reply content/managed-operations/migrate-gogrid-cloud-servers-to-racksapce-cloud.md]: Doh!

File: content/managed-operations/migrate-gogrid-cloud-servers-to-racksapce-cloud.md (hunk 3 above)
Comment (catlook) [reply content/managed-operations/migrate-gogrid-cloud-servers-to-racksapce-cloud.md]: won't work until PR merges (I think).  I'll test after pub to make sure!

File: content/managed-operations/migrate-gogrid-cloud-servers-to-racksapce-cloud.md (hunk 2 above)
Comment (catlook) [reply content/managed-operations/migrate-gogrid-cloud-servers-to-racksapce-cloud.md]: Leaving comma in place for now. CYR says Grammarly is good with the comma as is. From our style guide: 99% of occurrences of “such as” in the SG either have a comma before them, or a left paren, which I think of as kind of a comma.

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-c76b968bdc/how-to/)

**Pull request merged**